apps/salesforce-commerce-cloud/src/components/dialog/SearchBar.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useState } from 'react';
```

Comment:
Could we create a SearchBar folder, and break these out into separate files? This one is a big one! Also similar comments as earlier with consistently destructing props when you're using them in the component (eg. SearchBar doesn't need it because it's just passing them, but the subcomponents need them)